Guardian/Networking/ClientInfo.swift

```diff
@@ -29,15 +29,17 @@ private let infoEntryName = "CFBundleShortVersionString"
 struct ClientInfo: Codable, Equatable {
     let name: String
     let version: String
+    let env: Auth0TelemetryInfo?
```

Comment:
I don't see any references to `env` outside of this struct. What is its purpose?

Reply:
this is an addition for serialized ClientInfo (`base64(json(ClientInfo))`) which is getting put in the header value, datawarehouse can give us statistics on values inside which is exactly what we aim for here